libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -183,6 +196,30 @@ class AP_Motors {
     // convert input in 0 to +1 range to pwm output
     int16_t calc_pwm_output_0to1(float input, const SRV_Channel *servo);
 
+    // Apply rate of change limiting to an input
+    // max_rate is the maximum allowed rate of change as a percentage of the range per second
+    // dt is the time delta in seconds between calls
+    void limit_value_slew_rate(float& previous, float& current, float range, float max_rate, float dt) {
```

Comment:
The `limit_demand_slew_rate` method is already big to be in the header, but this one is huge. Please move it to the cpp file.